src/coreclr/vm/eeconfig.cpp

```diff
@@ -769,6 +769,11 @@ HRESULT EEConfig::sync()
 #if defined(FEATURE_GDBJIT_FRAME)
     fGDBJitEmitDebugFrame = CLRConfig::GetConfigValue(CLRConfig::INTERNAL_GDBJitEmitDebugFrame) != 0;
 #endif
+
+#if defined(FEATURE_CACHED_INTERFACE_DISPATCH) && defined(FEATURE_VIRTUAL_STUB_DISPATCH)
+    fUseCachedInterfaceDispatch = CLRConfig::GetConfigValue(CLRConfig::INTERNAL_UseCachedInterfaceDispatch) != 0;
```

Comment:
Do we want to generate a warning or anything if you try to enable it in a build that doesn't support it?

Reply:
Naw, this can only happen in a checked/debug build and that won't have any interesting impact.